submissions/t-jay-777/Hooli-style Popup/style.less

```diff
@@ -0,0 +1,195 @@
+@width: 100vw;
+
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+body {
+	color: black;
+	font: 'Arial', sans-serif;
+}
+.container {
+	width: @width;
+	margin: 0 auto;
+}
+
+.nav {
+	.nav-menu {
+		display: flex;
+		flex-wrap: wrap;
+		flex-direction: row;
+		justify-content: flex-end;
+		margin: 15px;
+		li {
+			display: flex;
+			justify-content: center;
+			list-style-type: none;
+			margin: 1em;
+			a img:hover {
+				outline: 1px sold #00CED1;
+			}
+		}
+		a {
+			text-decoration: none;
+			color: #000000;
+		}
+		a:hover {
+			text-decoration-line: underline;
+		}
+		a img {
+			height: 1.5em;
+		}
+		.nav-submenu a img:hover {
+			outline: 1px solid #00CED1;
+		}
+	}
+	
+}
+
+.popup-menu-1::before {
+	content: '';
```

Comment:
Fix view and position of this element. Also, add padding to the popup to avoid overlapping border on hover by 'triangle' 
[Image](https://prnt.sc/qi5onr)